Helpers creation

FacundoPucheta opened the pull request:
New helpers added(4) on (/src/helpers):

- isInsidePolygon.js
- standardE30_2003.js | with small fixed value on lines 51 - 54.
- standardE30_2015.js 
- standardE30_2015Spec.js

FacundoPucheta committed:
feat(/src/helpers): new helper created

New helper added: isInsidePolygon

FacundoPucheta committed:
feat(/src/helpers): new helper created #2

New helper added: standardE30_2003

manduinca commented on src/helpers/isInsidePolygon.js:
```diff
@@ -0,0 +1,32 @@
+  
+return c;
+
+
```
Why three empty lines? I think one line is ok.

manduinca commented on src/helpers/e30_2003.js:
```diff
@@ -0,0 +1,74 @@
+  const lat = location_data.latitude;
+  const long = location_data.longitude;
+
+  const polygonsFound = await Polygon.findAll({
```
`foundPolygons`?

manduinca commented on src/helpers/e30_2003.js:
```diff
@@ -0,0 +1,74 @@
+    polygons.push(p.points.split("|"));
+  }
+
+  const coordinates = await Point_polygon.findAll({
```
maybe: `raw_coordinates`

manduinca commented on src/helpers/e30_2003.js:
```diff
@@ -0,0 +1,74 @@
+  const coordinates = await Point_polygon.findAll({
+    where: { type: 3 },
+  });
+  let coordenadas = [];
```
maybe: `coordinates`

FacundoPucheta committed:
feat(/src/helpers): new helper created #3

New helper added: standardE30_2015

FacundoPucheta committed:
feat(/src/helpers): new helper created #4

New helper added: standardE30_2015Spec
Helper standardE30_2003 fixed: line 51 and 54. (value changed)

manduinca reviewed (dismissed):
Only `interpolar` is in spanish.

manduinca reviewed (approved):
LGTM!
